sdk/ml/azure-ai-ml/azure/ai/ml/operations/_registry_operations.py

```diff
@@ -52,14 +52,19 @@ def __init__(
         self._init_kwargs = kwargs
 
     #@ monitor_with_activity(logger, "Registry.List", ActivityType.PUBLICAPI)
-    def list(self) -> Iterable[Registry]:
+    def list(self, *, scope: str = "resource_group") -> Iterable[Registry]:
         """List all registries that the user has access to in the current
-        resource group.
+        resource group or subscription.
 
+        :param scope: scope of the listing, "resource_group" or "subscription", defaults to "resource_group"
+        :type scope: str, optional
         :return: An iterator like instance of Registry objects
         :rtype: ~azure.core.paging.ItemPaged[Registry]
         """
-
+        if scope == "subscription":
```

Comment:
we need to guard against users giving us `subscription` in any casing arrangement. Also, consider making "subscription" a constant
```suggestion
        if scope.lower() == "subscription":
```